tdrs-backend/tdpservice/search_indexes/admin/filters.py

```diff
@@ -61,9 +61,23 @@ def _get_lookup_choices(self, request):
         else:
             type_query = Query(type=STT.EntityType.STATE) | Query(type=STT.EntityType.TERRITORY)
             queryset = queryset.filter(type_query)
-
         return (queryset.distinct().order_by('name').values_list('name', flat=True))
 
+    def lookups(self, request, model_admin):
+        """Available options in dropdown."""
+        listing_record_type = [i for i in str(request.path).lower().split('/') if i not in ['/', '']][-1]
+        options = []
+        if 'tribal' in listing_record_type:
+            objects = STT.objects.filter(type=STT.EntityType.TRIBE)
+        elif 'ssp' in listing_record_type:
+            objects = STT.objects.filter(ssp=True)
+        else:
```

Comment:
@raftmsohani does this reflect territories too TANF search indices?

Reply:
Good catch, I will have to revise this